modules/openapi-generator/src/main/resources/python/python-experimental/api_test.mustache

```diff
@@ -0,0 +1,37 @@
+# coding: utf-8
+
+{{>partial_header}}
+
+from __future__ import absolute_import
+
+import unittest
+
+import {{packageName}}
+from {{apiPackage}}.{{classVarName}} import {{classname}}  # noqa: E501
+from {{packageName}}.rest import ApiException
+
+
+class {{#operations}}Test{{classname}}(unittest.TestCase):
+    """{{classname}} unit test stubs"""
+
+    def setUp(self):
+        self.api = {{apiPackage}}.{{classVarName}}.{{classname}}()  # noqa: E501
```

Comment:
Can we spread this over two lines so we don't need the `# noqa: E501`?

Reply:
Done, but it will not work when the three variables have very large names